frontend/src/components/GoesData/FetchTab.tsx

```
@@ -31,294 +50,456 @@ function isDateInRange(dateStr: string, avail: SatelliteAvailability): boolean {
+      )}
+
+      {/* Confirmation modal */}
+      {showConfirm && (
+        <dialog
+          open
+          className="fixed inset-0 z-50 flex items-center justify-center bg-transparent p-4 m-0 w-full h-full max-w-none max-h-none [&::backdrop]{bg-black/50}"
+          onClick={(e) => { if (e.target === e.currentTarget) setShowConfirm(false); }}
+          onClose={() => setShowConfirm(false)}
+          aria-labelledby="confirm-title"
+        >
+          <div
```

Biome (2.3.14)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)